src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable jsx-a11y/label-has-for */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class TodoItem extends Component {
+  handleClickCheckBox = ({ target }) => {
+    const id = +target.id[target.id.length - 1];
+    const { changeCompleted } = this.props;
+
+    changeCompleted(id);
+  }
+
+  handleClickButtonDestroy = ({ target }) => {
+    const idInput = target.parentNode.children[0].id;
+    const idTodo = +idInput[idInput.length - 1];
+    const { removeTodos } = this.props;
+
+    removeTodos([idTodo]);
+  }
+
+  handleLabelClick = (event) => {
+    event.preventDefault();
+  }
+
+  handleLableDoubleClick = ({ target }) => {
+    const parentLi = target.closest('li');
+    const newInput = document.createElement('input');
```

Comment:
we don't use direct DOM interaction, remove this and rewrite with pure react

Reply:
> we don't use direct DOM interaction, remove this and rewrite with pure react

refactored